packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -104,13 +105,17 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     argParser.addOption(
       'export-method',
       defaultsTo: 'app-store',
-      allowed: <String>['app-store', 'ad-hoc', 'development', 'enterprise'],
+      allowed: <String>['app-store', 'app-store-connect', 'ad-hoc', 'release-testing', 'development', 'debugging', 'enterprise'],
       help: 'Specify how the IPA will be distributed.',
       allowedHelp: <String, String>{
-        'app-store': 'Upload to the App Store.',
+        'app-store': 'Upload to the App Store. For XCode <= 15.3.',
```

Comment:
XCode is cased "Xcode". Someday I'm going to make a linter for this...
```suggestion
        'app-store': 'Upload to the App Store. For Xcode <= 15.3.',
```